src/query_graph/field_set.rs

```diff
@@ -19,36 +47,25 @@ pub(super) fn parse_field_set(
         value.as_str(),
         "field_set.graphql",
     )?;
-    merge_selection_sets(&[&field_set.selection_set])
+    Ok(field_set.into_inner().selection_set)
 }
 
 pub(super) fn merge_selection_sets(
```

Comment:
since we define `NormalizedSelectionSet` in `operation.rs` -> should we move this function (and `equal_selection_sets`) over there?

Reply:
@dariuszkuc 
Makes sense, moved the functions over.